src/main/resources/schemas/schema-form.json

```diff
@@ -34,6 +34,12 @@
       "default": 2000,
       "minimum": 0
     },
+    "releaseCache": {
+      "title": "Release the cache when on stop the API ?",
```

Comment:
`Release the cache when on stop the API ?` => `Release the cache when API is stopped?`